src/test/policies/pOLY.t.sol

```diff
@@ -163,6 +162,18 @@ contract pOLYTest is Test {
         }
     }
 
+    function _checkTerms(
```

Comment:
NB: This was updated to help with getting coverage working - ie avoid stack too deep.